include/zephyr/storage/stream_flash.h

```diff
@@ -65,8 +65,8 @@ struct stream_flash_ctx {
 #ifdef CONFIG_STREAM_FLASH_ERASE
 	off_t last_erased_page_start_offset; /* Last erased offset */
 #endif
+	size_t write_block_size;	/* Offset/size device write alignment */
 	uint8_t erase_value;
```

Comment:
erase_value also needs to be size_t, this is from the zephyr/dts/arm/nxp/nxp_lpc55S6x_common.dtsi
![image](https://github.com/user-attachments/assets/255da590-7163-4886-9431-295cbfb11ce5)

Reply:
erase value is the single byte of what the erased data is, it should be a single byte and is typically 0xff for flash

Reply:
Yes, you are correct. Thank you, please ignore my comment.